From zaitcev@redhat.com  Tue Jun 23 14:54:10 2009
From: Pete Zaitcev <zaitcev@redhat.com>
Date: Wed, 10 Jun 2009 15:21:24 -0600
Subject: USB: usbmon: drop Kconfig defaults
To: linux-usb@vger.kernel.org, <greg@kroah.com>
Cc: Paul Bolle <pebolle@tiscali.nl>, Alan Stern <stern@rowland.harvard.edu>, David Brownell <david-b@pacbell.net>, zaitcev@redhat.com
Message-ID: <20090610152124.ee8df387.zaitcev@redhat.com>


These statements seem to be unnecessary. No idea why, but I built all
possible configurations and everything gets built just as before.

It's an old patch that popped from discussion with Paul in November 2008.
Obviously not a very high priority but better late than never.

Signed-off-by: Pete Zaitcev <zaitcev@redhat.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/mon/Kconfig |    4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

--- a/drivers/usb/mon/Kconfig
+++ b/drivers/usb/mon/Kconfig
@@ -5,11 +5,9 @@
 config USB_MON
 	tristate "USB Monitor"
 	depends on USB
-	default y if USB=y
-	default m if USB=m
 	help
 	  If you select this option, a component which captures the USB traffic
 	  between peripheral-specific drivers and HC drivers will be built.
 	  For more information, see <file:Documentation/usb/usbmon.txt>.
 
-	  If unsure, say Y (if allowed), otherwise M.
+	  If unsure, say Y, if allowed, otherwise M.
